openapi.yml

```diff
@@ -142,6 +142,12 @@ paths:
           required: true
           schema:
             type: string
+        - name: avoid_vlan
```

Comment:
This query arg hasn't been renamed yet based on your last refactoring

Reply:
Changed from `avoid_vlan` to `try_avoid_same_s_vlan`